resources/assets/less/bem/osu-page-header-v2.less

```diff
@@ -24,7 +24,7 @@
   padding: 20px;
   display: flex;
   flex-direction: column;
-  justify-content: flex-start;
+  justify-content: center;
```

Comment:
this pushes the header down further than it should be on desktop, I'd put it in a `@media @mobile` block instead